### PR TITLE
Pass in auth token

### PR DESCRIPTION
**Pull request by pkerpedjiev**
Add the ability to pass in an auth_token via the `auth_token` kwarg.

Pardon the formatting. My IDE did it 😬. If it's super annoying I can submit another PR with just formatting changes. Or try to submit with the previous formatting and just the functional change.

I marked the only two lines of functional changes.

### CODE REVIEW

File: higlass_widget/widget.js [hunk 1]
```diff
@@ -7,34 +7,44 @@ import hglib from "https://esm.sh/higlass@1.12?deps=react@17,react-dom@17,pixi.j
...
+  let [x, xe] = xDomain;
+  let [y, ye] = yDomain;
+  return [x, xe, y, ye];
 }
 
 export async function render({ model, el }) {
-	let viewconf = JSON.parse(model.get("_viewconf"));
-	let api = await hglib.viewer(el, viewconf);
+  let viewconf = JSON.parse(model.get("_viewconf"));
+  let auth_token = model.get("auth_token");
```
Comment (pkerpedjiev) [inline higlass_widget/widget.js:17]: Functional change

File: higlass_widget/widget.py [hunk 2]
```diff
@@ -10,6 +10,8 @@ class HiGlassWidget(anywidget.AnyWidget):
     _esm = pathlib.Path(__file__).parent / "widget.js"
     _css = "https://esm.sh/higlass@1.12/dist/hglib.css"
     _viewconf = t.Unicode("null").tag(sync=True)
+    
+    auth_token = t.Unicode().tag(sync=True)
```
Comment (pkerpedjiev) [inline higlass_widget/widget.py]: Functional change

**Review by manzt — COMMENTED** [review]
> Pardon the formatting. My IDE did it 😬.

Lol, no worries. We don't have any linting/formatting setup and 2 spaces over tabs is more standard in JS land.

What do we think about having an `options` dict (rather than a separate trait for each option):

```python
HiGlassWidget(conf.dict(), options={"authToken": "XXXX", "theme": "dark" })
```

Second, since migrating to `anywidget`, I think its probably way nicer to move this to `higlass-python`? Thoughts?

**Comment by pkerpedjiev** [comment]
Fully support 👍

**Pull request closed without merging**